pennylane/devices/default_clifford.py

```diff
@@ -245,18 +336,28 @@ def __init__(
         shots=None,
         check_clifford=True,
         tableau=True,
+        target_states=None,
         seed="global",
         max_workers=None,
     ) -> None:
+        if not has_stim:
+            raise ImportError(
+                "This feature requires stim, a fast stabilizer circuit simulator. "
+                "It can be installed with:\n\npip install stim"
+            )
```

Comment:
Maybe a bit late to the game, but why do we call this device `default.clifford` if it is a PennyLane interface to `stim`? _Could_ be perceived as adorning ourselves with borrowed plumes, not sure, the device doc first line states and links stim so could be fine 🤔 CC @trbromley